Github-Issue-Guide.md

```diff
@@ -0,0 +1,68 @@
+# Filing an Issue on Oppia's GitHub: A Step-by-Step Guide for Non-Developers
```

Comment:
You don't need a title. GitHub will automatically set the title to be the filename without the hyphens

Reply:
Thanks, Done.